application/__init__.py

```
@@ -21,22 +21,26 @@
 import os
 import sys
 
+import application.paths
```

Ruff (0.8.2)

`application.paths` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)